### PR TITLE
Fix #1829: Added warning supression for new versions of jshint

### LINKED ISSUE #1829: npm test for new deps failed because of "A constructor name should start with an uppercase letter"
I was trying to update node deps for the project and CI Travis shows me an [error](https://travis-ci.org/mozilla/thimble.mozilla.org/builds/208818821)

screenshot 2017-03-07 17 55 13

According to it, the next [function](https://github.com/mozilla/thimble.mozilla.org/blob/master/server/lib/environment.js)
```js
var habitat = require('habitat');
habitat.load(require('path').resolve(__dirname, '../../.env'));
var env = new habitat();
``` 
is wrong, because jshint expects that an identifier following the new operator(when the newcap option is set to true)  should start with an upperCase.

### PR DESCRIPTION
**Pull request by omytryniuk**
Here is the fix of the issue that happens with [deps update](https://github.com/mozilla/thimble.mozilla.org/pull/1828). 
This request updates deps, and the newest version of jshint, treats this code as an error, instead of just as a warning. 
```js
var habitat = require('habitat');
habitat.load(require('path').resolve(__dirname, '../../.env'));
var env = new habitat();
```
There is anything wrong with this code. That's why the easist solution to pass new updates of jhint, is to supress the warning. In addition, this syntax check will be deprecated in the next version of jshint.

### CODE REVIEW

File: server/lib/environment.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/*jshint newcap:false */
+
```
Comment (humphd) [inline server/lib/environment.js]: Can you remove this blank line, since it relates directly to line 3.

**Comment by humphd** [comment]
One small nit to fix, and this is good to go.

File: server/lib/environment.js (hunk 1 above)
Comment (omytryniuk) [inline server/lib/environment.js]: @humphd Sure! Thank you. I just saw in  the previous examples with jshint directives a blank line was left and that is why I left it; however I will fix it. 
Just a small question. 
Here:
```js
/*jshint newcap:false */
```
Should I keep the extra space between false and * (same as it is used by jshint developers), or just get rid of the space? Thank you

Comment (humphd) [inline server/lib/environment.js]: If it were me, I'd but a space before/after both `*` but it's all optional.

**Comment by humphd** [comment]
Thanks for researching and fixing this.  You should rebase #1828 on top of master now that this has landed.

**Comment by omytryniuk** [comment]
You are welcome, @humphd. I will do rebase. Thank you for your help!

**Pull request merged**